### DISCUSSION TITLE
Couple stupid questions

### DISCUSSION THREAD

**Discussion by zenorach** [keeweb/keeweb#2167] [Q&A / Support]
So I had a couple of questions

1) Is there some way to turn off the splash page telling you the browser is trying to make a connection to keeweb as long as keeweb is unlocked? The number of times that I lose that popup and then go "oh yeah" and have to find it is annoying. It'd be great if that was a setting within the application so we can choose. The splash screen I'm talking about is
`Extension data exchange`

2) Is there a way to change the "dropbox" location. So right now I use both keeweb and keepass (mostly because so far I haven't found a great keeweb my my android that links in as well). So when I select dropbox it goes to 
`/<dropbox_home>/Apps/Keeweb`
 ...... but my keepass goes to 
`/<dropbox_home>/Apps/Keepass/ `
Is there a way to easily modify keeweb so it'll share a home with keepass? because that'd make my life just a touch more convenient

**Comment by Aetherinox** [comment]
@zenorach Sorry for the delay, mid-update right now, been busy. And don't worry, no question is stupid.

1. Can you screenshot the page you're speaking of? I don't believe I've seen that page yet; unless you're speaking of the browser page that pops up letting you know that you have successfully authenticated with the service.

2. If you open your **Advanced Settings** of KeeWeb, you should have a dropdown for Dropbox which allows you to select an alternative location. However, it will request additional info depending on the option you select.

![HVWz2lXKMb](https://github.com/keeweb/keeweb/assets/118329232/3618ee67-7a8f-45bc-ba63-83723cf7c2cf)

There are two additional options in the dropbox, including **Full Dropbox or any folder**, which is where you specify only a folder, without the need for a dropbox API.

![OU6ylhIbG4](https://github.com/keeweb/keeweb/assets/118329232/78455523-3798-4b76-a8dc-b5095937d3a6)

Then after you sync your vault db file the next time, you should see an additional folder in your Dropbox:

![8yzRPZkuvx](https://github.com/keeweb/keeweb/assets/118329232/ae1d98b6-70cf-4b28-9fd2-4aad34eacf36)

Now, I am new to the development team. I've had to cram years worth of development into a few weeks. So I have not done a lot with this feature. The setting **Full Dropbox on any folder** works on the beta site at: https://beta.keeweb.info/. I am unsure if the desktop version has this feature, or if it requires the browser extension as well so that it can communicate.

I'll have to look through the code and see.

> [!IMPORTANT]
> Before you change the vault settings of your personal vault, back it up to a local directory. There are a lot of things that are outdated in that release, and I don't want you losing your vault in case there's a bug. Do that before changing the dropdown settings.
